dashboard/database.py

```diff
@@ -23,64 +23,63 @@ def get_connection() -> Connection:
 
 
 @st.cache_data(ttl="1hr")
-def get_popular_tracks(_conn: Connection, n: int = 5) -> pd.DataFrame:
-    """Returns the N most sold tracks in the database."""
+def get_popular_tracks(_conn: Connection, timeframe: str) -> pd.DataFrame:
+    """Returns the 5 most sold tracks in the database."""
 
     print("Collating most popular tracks...")
-
-    query = """
+    query = f"""
         SELECT T.title, A.name, COUNT(track_purchase_id) AS copies_sold, T.url as url
-        FROM track_purchase AS PT
```

Comment:
Yeah that does make sense, makes me wonder why it was reversed in the first place